LocalSettings.php

```diff
@@ -2607,12 +2607,24 @@
 	// Linter
 	'wgLinterSubmitterWhitelist' => [
 		'wmgUseLinter' => [
```

Comment:
Why is this `wmgUseLinter` and not `default`?

Reply:
Because that makes it only be used when linter is enabled.

Reply:
Hmm, I'm unfamiliar with this syntax for wgConf, but it does appear that it works.

Reply:
It's something that was added not very long ago.